amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -406,21 +406,22 @@ type UserStakes {
 type StakerRewards {
   address: String!
   rewards: MotionStakeValues!
+  isClaimed: Boolean!
 }
 
 type MotionData {
-  motionId: String!
+  motionId: String! # to ensure uniqueness, we format as: chainId-votingRepExtnAddress_nativeMotionId
```

Comment:
Introducing this format for motion ids is useful in the block ingestor when fetching a motion from the db (since native motion ids are not unique across voting rep installations) and also in the claimReward saga, when we need to check which motions have outstanding claims.

Reply:
https://github.com/JoinColony/block-ingestor/pull/42